src/node.cc

```diff
@@ -2834,6 +2837,13 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "traceDeprecation", True(env->isolate()));
   }
 
+  // --warn-on-sync
+  if (warn_on_sync) {
+    READONLY_PROPERTY(process, "warnOnSync", True(env->isolate()));
```

Comment:
I'm just following the current implementation for all other properties of this type, e.g. `process.traceDeprecation`.